packages/gestalt/src/Tooltip/InternalTooltip.js

```diff
@@ -60,7 +68,7 @@ type Props = {|
   idealDirection?: 'up' | 'right' | 'down' | 'left',
   inline?: boolean,
   link?: Node,
-  text: string,
+  text: string | Element<typeof Text>,
```

Comment:
Do we wanna add an example like https://gestalt.pinterest.systems/web/toast#Message

Does this require any guidelines?

Reply:
removed to not use string array